evalml/utils/woodwork_utils.py

```diff
@@ -0,0 +1,112 @@
+
+import numpy as np
+import pandas as pd
+import woodwork as ww
+
+numeric_and_boolean_ww = [ww.logical_types.Integer, ww.logical_types.Double, ww.logical_types.Boolean]
```

Comment:
Maybe not too important for now, but note that this could fail to capture everything if a user defines a custom numeric or boolean type in the TypeSystem...could be something like WholeNumber or Currency.

Reply:
Yes, you're right! I filed https://github.com/alteryx/evalml/issues/1810 to track this, thank you!